clustered/src/main/java/consumer/Monitoring.java

```diff
@@ -0,0 +1,40 @@
+package consumer;
+
+import static commons.Config.BY;
+import static commons.Config.REBOOT_ADDRESS;
+import static commons.Config.STATUS;
+
+import java.util.logging.Logger;
+
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Future;
+import io.vertx.core.VertxOptions;
+import io.vertx.core.json.JsonObject;
+import io.vertx.reactivex.core.Vertx;
+
+public class Monitoring extends AbstractVerticle {
+
+   private final Logger logger = Logger.getLogger(Monitoring.class.getName());
+
+   @Override
+   public void start(Future<Void> startFuture) throws Exception {
+      logger.info("Reboot monitor started");
+      vertx.eventBus().consumer(REBOOT_ADDRESS, message -> {
+         JsonObject reboot = (JsonObject) message.body();
```

Comment:
same here, use a typed consumer method and you don't need to cast

Reply:
ok